pywbem_mock/_dmtf_cim_schema.py

```diff
@@ -87,15 +90,105 @@
 __all__ = ['DMTFCIMSchema']
 
 
+def build_schema_mof(schema_classes, schema_mof_pragma_file):
```

Comment:
We currently have three ways to name the parameter for the class names in the various functions: `schema_classes`, `schema_classnames`, and `class_names`. For consistency, I suggest to name them all `class_names`. I have according comments on each case.
So please rename the `schema_classes` parameter to `class_names`, for consistency.